File: 백준/Bronze/2577. 숫자의 개수/숫자의 개수.py
import sys
input = sys.stdin.readline

a = int(input())
b = int(input())
c = int(input())

result = a*b*c
result = str(result)
num_dict = {'0':0, '1':0, '2':0, '3':0, '4':0, '5':0, '6':0, '7':0, '8':0, '9':0}
for i in result:
    num_dict[i] += 1

for i in num_dict.values():
    print(i)

File: 백준/Bronze/2577. 숫자의 개수/숫자의 개수.js
const fs = require("fs");
const filePath = process.platform === "linux" ? "/dev/stdin" : "./input.txt";
const input = fs
  .readFileSync(filePath)
  .toString()
  .trim()
  .split("\n")
  .map(Number);

let result = input[0] * input[1] * input[2];

result = String(result);
const obj = {};

for (let i = 0; i < 10; i++) {
  obj[i] = 0;
}
for (let i = 0; i < result.length; i++) {
  let num = Number(result[i]);
  obj[num] += 1;
}

for (let i = 0; i < 10; i++) {
  console.log(obj[i]);
}
